### ISSUE TITLE
Generating image using the CLI: always gets stuck at 69.29%

### ISSUE THREAD

**Issue by skerit** [Querz/mcaselector#289]
**Describe the bug**

When I try to generate an image of my world it always gets stuck at 69.29%, which is **not** nice.
I've tried to increase the heap size, but that doesn't change the outcome.

**To Reproduce**

1. Generate a chunk CSV selection
2. Go to the terminal
3. Execute `mcaselector --region world/region/ --mode image --selection chunks-keep-feather-5.csv --output test.png`

**Expected behavior**

I expect it to generate everything

**Screenshots and other files**

The world is quite big (6.3 gigabytes) and the CSV is 1.7 megabytes (127794 lines)
(So if you need any file, please ask)

Generating the image in the GUI works fine though:

![chunks-keep-feather-5-small](https://user-images.githubusercontent.com/755212/144232499-5e2a918f-1a4b-41c5-bc79-1f6e5d0705fb.png)

Another thing that pops up in the CLI mode is this error:

```
Exception in thread "processPool-thread-119" java.lang.RuntimeException: Internal graphics not initialized yet
	at javafx.graphics/com.sun.glass.ui.Screen.getScreens(Screen.java:68)
	at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.getScreens(QuantumToolkit.java:765)
	at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.getMaxRenderScale(QuantumToolkit.java:792)
	at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.loadImage(QuantumToolkit.java:801)
	at javafx.graphics/javafx.scene.image.Image.loadImage(Image.java:1073)
	at javafx.graphics/javafx.scene.image.Image.initialize(Image.java:819)
	at javafx.graphics/javafx.scene.image.Image.<init>(Image.java:641)
	at net.querz.mcaselector.io.job.SelectionImageExporter$ExportSelectionImageProcessJob.execute(SelectionImageExporter.java:106)
	at net.querz.mcaselector.io.job.ProcessDataJob.run(ProcessDataJob.java:70)
	at net.querz.mcaselector.io.JobHandler$WrapperJob.run(JobHandler.java:237)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
```

**Environment (please complete the following information):**
- OS: Linux (Manjaro)
- Java version: 17.0.1
- Version of MCA Selector: 1.17

**Additional context**

- Minecraft version used to generate affected parts of your world: 1.17.1
- Mods used that change Minecraft save files or add blocks to the game: Custom mod
- Previously used programs to alter the world files: none

**Comment by baraclese** [comment]
I had the same problem. I created an image once, then I modified the world and the query and tried to create the image again. That is when the exception occurred. Deleting the cache at .cache/mcaselector solved the problem.

**Comment by Querz** [comment]
Fixed in [release 1.17.2](https://github.com/Querz/mcaselector/releases/tag/1.17.2)
